rpki-validator/src/main/java/net/ripe/rpki/validator3/config/ValidationConfig.java

```diff
@@ -39,16 +39,30 @@
 @ConfigurationProperties("rpki.validator")
 public class ValidationConfig {
 
-    @Getter @Setter
+    @Getter
+    @Setter
     private boolean strictValidation;
 
-    @Getter @Setter
+    /**
```

Comment:
Perhaps this name isn't very good.
`earlyWarningValidation`?

Reply:
Yeah, makes sense.